src/main/java/com/g4s8/ghman/web/PsNumericFlag.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.web;
+
+import java.io.IOException;
+import org.takes.Request;
+import org.takes.Response;
+import org.takes.facets.auth.Identity;
+import org.takes.facets.auth.Pass;
+import org.takes.misc.Opt;
+import org.takes.rq.RqHref;
+
+/**
+ * Numeric Flag identity extraction.
+ * @since 1.0
+ * @todo #4:30min Replace the call to static method String.format
```

Comment:
@samihibrahim please re-read better my review here: https://github.com/g4s8/ghman/pull/13#issuecomment-499862708 there is still multiple todo for the same task. Don't forget each todo you add will result in a new task. Honestly it would have taken you less time to just fix it instaad of adding those todos...

Reply:
@victornoel again sorry for being dense. Fixed the Todos typo but I as the original task was only related to the refactoring of logic, I think it replacing these functions should have their own todos.